charts/gardener-extension-provider-openstack/templates/deployment.yaml

```diff
@@ -103,7 +103,7 @@ spec:
         {{- end }}
       affinity:
         podAntiAffinity:
-          requiredDuringSchedulingIgnoredDuringExecution:
+          preferredDuringSchedulingIgnoredDuringExecution:
```

Comment:
This could regress the HA mode of the extension that also serves webhooks - needed for GEP-20.
cc @timuthy @unmarshall

Reply:
As @ialidzhikov said, this is not compatible with what is desired in an HA scenario where we want to forcefully distribute the pods across nodes/zones.
If you want to support this soft affinity with multiple replicas, then I suggest to enhance the chart values by an optional affinity setting that can be passed.
Even better, use https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/ instead of `podAntiAffinity`.

Reply:
I agree TSC would be a better option here.